### ISSUE TITLE
Request to release 0.4.1

### ISSUE THREAD

**Issue by etipton** [zweilove/css_splitter#49]
This gem is awesome and saved me a lot of hassle. However, I definitely got burned by the "Empty *_split2.css file" gotcha mentioned in the README. I didn't know what debug: true meant or ?body=1 or any of that and had to do some digging to figure out what was going on. Currently the solution for me is:

```ruby
# For css_splitter, using a ref b/c that commit uses debug: false by default for _splitN stylesheets
gem 'css_splitter', git: 'git://github.com/zweilove/css_splitter.git', ref: 'a45245fb2114c90f22d2a68b10c70eb1ac641e7d'
```

but obviously I would rather point to v0.4.1

**Comment by jhilden** [comment]
I just released version `0.4.1`

**Comment by etipton** [comment]
Awesome, thanks!
